src/config/config.ts

```diff
@@ -21,19 +21,21 @@ const fornoApiKey = process?.env?.NEXT_PUBLIC_FORNO_API_KEY || '';
 const celoscanApiKey = process?.env?.NEXT_PUBLIC_CELOSCAN_API_KEY || '';
 const infuraApiKey = process?.env?.NEXT_PUBLIC_INFURA_API_KEY || '';
 const upstashKey = process?.env?.UPSTASH_KEY || '';
+const useAlfajores = process?.env?.NEXT_PUBLIC_USE_ALFAJORES === '1' || false;
 
-export const fornoRpcUrl = `${links.forno}?apikey=${fornoApiKey}`;
-export const infuraRpcUrl = `${links.infura}/${infuraApiKey}`;
+export const fornoRpcUrl = `https://forno.celo.org?apikey=${fornoApiKey}`;
```

Comment:
links are now hardcoded as they were used only here and keeping them in `links` package was causing circular dependency config <-> links

Reply:
cool